resources/aws/backoff.go

```diff
@@ -25,3 +28,9 @@ func NewCustomExponentialBackoff() *backoff.ExponentialBackOff {
 
 	return b
 }
+
+func NewNotify(logger micrologger.Logger, operationName string) func(error, time.Duration) {
+	return func(err error, delay time.Duration) {
+		logger.Log("error", fmt.Sprintf("%s failed, retrying with delay %.0fm%.0fs: %v", operationName, delay.Minutes(), delay.Seconds(), errgo.Details(err)))
```

Comment:
Not really. `delay.String()` shows milliseconds as well:
https://play.golang.org/p/BLbNfkpaHh